tests/conftest.py

```diff
@@ -57,3 +73,57 @@ def new_payment_object(self, sender=LocalAccount.generate(), receiver=LocalAccou
     def new_sender_payment_command(self):
         payment = self.new_payment_object()
         return offchain.PaymentCommand(my_actor_address=payment.sender.address, payment=payment, inbound=False)
+
+    def new_funds_pull_pre_approval_object(self, biller=LocalAccount.generate(), receiver=LocalAccount.generate()):
+        address = identifier.encode_account(
+            receiver.account_address,
+            identifier.gen_subaddress(),
+            self.hrp(),
+        )
+
+        biller_address = identifier.encode_account(
+            biller.account_address,
+            identifier.gen_subaddress(),
+            self.hrp(),
+        )
+
+        return FundPullPreApprovalObject(
+            funds_pull_pre_approval_id=str(uuid.uuid4()),
+            address=address,
+            biller_address=biller_address,
+            scope=FundPullPreApprovalScopeObject(
+                type="consent",
+                expiration_timestamp=int(time.time()) + 30,
+                max_cumulative_amount=ScopedCumulativeAmountObject(
+                    unit="week",
+                    value=1,
+                    max_amount=CurrencyObject(amount=1_000_000_000_000, currency=testnet.TEST_CURRENCY_CODE),
+                ),
+                max_transaction_amount=CurrencyObject(amount=1_000_000, currency=testnet.TEST_CURRENCY_CODE),
+            ),
+            status=FundPullPreApprovalStatus.valid,
+            description="test",
+        )
+
+    def new_funds_pull_pre_approval_command(self):
+        funds_pull_pre_approval = self.new_funds_pull_pre_approval_object()
+        return offchain.FundsPullPreApprovalCommand(
+            my_actor_address=identifier.encode_account(
+                LocalAccount.generate().account_address,
+                identifier.gen_subaddress(),
+                self.hrp(),
+            ),
+            funds_pull_pre_approval=funds_pull_pre_approval,
```

Comment:
I think this missing inbound field, and will fail, if no code is using this, may remove this function.

Reply:
Wow, great catch!
Fixed.